src/components/common/NewActionButton/NewActionButton.tsx

```diff
@@ -64,7 +64,7 @@ const MSG = defineMessages({
 // }
 
 const NewActionButton = (/* { colony, ethDomainId }: Props */) => {
-  const canInteractWithColony = useColonyContext();
+  const { canInteractWithColony } = useColonyContext();
```

Comment:
I noticed this issue, although, feel free to correct me if this was not the intended purpose.